Update CG example

fredvisser opened the pull request:
I neglected to update the CG example description and message topic name in the previous PR. This PR resolves that issue. 
In a future release, we'll remove this example, and leverage the CG API shipping examples instead.

rajsite commented:
A couple minor things:

1. Can we remove the cRIO target (if it makes sense to leave it that's fine too)
![image](https://user-images.githubusercontent.com/1588923/63124629-55034c80-bf71-11e9-94bc-dbf0be1ff491.png)

2. The connection cases have some typos in the comments
![image](https://user-images.githubusercontent.com/1588923/63124702-7bc18300-bf71-11e9-8efe-142fa01b0895.png)

![image](https://user-images.githubusercontent.com/1588923/63124731-95fb6100-bf71-11e9-9861-410b9a42a385.png)

![image](https://user-images.githubusercontent.com/1588923/63124770-a7446d80-bf71-11e9-8914-30c964c66b84.png)

fredvisser commented:
I've removed the RT target, as it adds unnecessary complication to a simple example. And fixed the egregious spelling errors in the block diagram comments.
